packages/mui-material/test/typescript/reactAugmentation/children.spec.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
```

Comment:
We shouldn't include this test since it's not related to MUI. It is related to augmentation in the react children types.